examples/pipeline_tacotron2/inference.py

```diff
@@ -0,0 +1,294 @@
+from functools import partial
+import argparse
+import os
+import random
+import sys
+
+import torch
+import torchaudio
+import numpy as np
+from torchaudio.prototype.tacotron2 import Tacotron2
+from torchaudio.models.wavernn import _MODEL_CONFIG_AND_URLS
```

Comment:
Can you put this into `_parse_args` so that it will be (somewhat) hidden?

Reply:
Good suggestion! I've moved it into `parse_args` [here](https://github.com/pytorch/audio/pull/1654/files#diff-190b61efc2a83079d005a5da48a9fde8b28a937a8a653a8b4ede3b792f742384R31)